libbpf-rs/src/probe.rs

```diff
@@ -0,0 +1,57 @@
+//! Probe for supported system features
+//!
+//! Note that running many of these probes in a short amount of time can cause
+//! the kernel to reach the maximal size of lockable memory allowed for the
+//! user, causing subsequent probes to fail. In this case, the caller may want
+//! to adjust that limit with setrlimit().
+
+use super::map::MapType;
+use super::program::ProgramType;
+use libbpf_sys::bpf_func_id;
+
+use crate::*;
```

Comment:
I understand there is precedent to the contrary here, but can we please stop using glob imports like that? It's fine for pulling in `super` definitions in a unit test module, but in the core library I'd prefer we list the imports explicitly, so that it's easier to follow where something is coming from or what it refers to. It also prevents naming conflicts from the get go. What do you think?

Also see note here: https://phaiax.github.io/mdBook/rustbook/ch07-03-importing-names-with-use.html#glob-imports-with-

Reply:
no prob, i just tried to follow existing style. will change